undertow/EAP7-832_Secure_Cookie.adoc

```diff
@@ -0,0 +1,51 @@
+= EAP7-832 Secure Cookie Handling
+:author:            Stuart Douglas
+:email:             sdouglas@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org
+
+== Overview
+
+The goal of this feature is to add an option to mark all cookies that are set on a secure connection with the
+secure attribute. This means that a cookie that is set over a secure connection will never be sent over an unsecure
```

Comment:
typo - `set` -> `sent`

Reply:
Both set and sent work here, but set is less ambiguous imho as it makes it clear it is referring to set-cookie.

Reply:
Oh, of course, you are right. In this case `set` is better. Keep it that way, then :)